.jenkins/Jenkinsfile

```diff
@@ -100,11 +144,21 @@ def win2016DebugCrossPlatform() {
         node {
             cleanWs()
             checkout scm
-            def oetoolsWincp = docker.build("oetools-wincp", "-f .jenkins/Dockerfile .")
-            oetoolsWincp.inside {
-                timeout(15) {
-                    sh './scripts/test-build-config -p SGX1FLC -b Debug --compiler=clang-7'
-                    stash includes: 'build/tests/**', name: 'linuxdebug'
+
+            docker.withRegistry(OETOOLS_REPO, OETOOLS_REPO_CREDENTIAL_ID) {
+                docker.image(OETOOLS_IMAGE).inside {
+                    timeout(15) {
+                        dir('build') {
+                            withEnv(["CC=clang-7","CXX=clang++-7"]) {
+                                sh """
+                                cmake .. -DCMAKE_BUILD_TYPE=Debug
+                                make
+                                OE_SIMULATION=1 ctest --output-on-failure
```

Comment:
I don't think this is supposed to be in simulation mode.

Reply:
And that's why we're getting rid of this script. I got confused (again) by the double negative of `-d` meaning "disable simulation" and lacking it meaning "enable simulation."